shared/reducers/pinentry.js

```diff
@@ -58,41 +49,28 @@ export default function(
           },
         }
       }
-      return state
-    default:
-      if (state.started && sessionID != null && isPinentryAction(action)) {
```

Comment:
old way: pass all actions to another thing that is basically a switch, new way explicilty just handle onCancel/onSubmit in this switch